mne/gui/tests/test_ieeg_locate_gui.py

```diff
@@ -0,0 +1,22 @@
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-clause
+
+import os.path as op
+import numpy as np
+
+import pytest
+
+import mne
+from mne.datasets import testing
+
+data_path = testing.data_path(download=False)
+subject = 'sample'
+subjects_dir = op.join(data_path, 'subjects')
+sample_dir = op.join(data_path, 'MEG', subject)
+raw_path = op.join(sample_dir, 'sample_audvis_trunc_raw.fif')
+fname_trans = op.join(sample_dir, 'sample_audvis_trunc-trans.fif')
+
+
+def test_ieeg_elec_locate_gui_display():
+	mne.gui.locate_ieeg(subject, subjects_dir=subjects_dir)
```

Comment:
can you micmic clicks ? I suspect this test leads to limited coverage no?

Reply:
Yes, hopefully that will be done soon, I've been sorting out the API (e.g. other PR) but this is next